docs/adding_a_new_graphql_microservice.md

```diff
@@ -31,7 +31,7 @@ Let's pretend we're mapping a GraphQL API called `Three Body` but provides its o
    import fetch from "node-fetch"
    import urljoin from "url-join"
 
-   import { middlewareLink } from "../lib/middlewareLink"
+   import { middlewareLink } from "lib/middlewareLink"
```

Comment:
Nesting in `/v1` caused several of these relative paths to break. I've update them to use the root import provided via ts.